app/views/selfservicetimetopay/core/sign_in_question.scala.html

```diff
@@ -0,0 +1,30 @@
+@*
+Self Service Time to Pay eligibility page - ask user what kind of tax debts they have to pay
```

Comment:
This page still have information from what it was copied from, this need to be cleaned up